AGENTS.md

```
@@ -0,0 +1,51 @@
+2. Perform handler operations
+3. Update check-run with status `completed` and appropriate conclusion
+
+### Registration
+- Add handler to `CONFIG_SPEC` in `src/auto-me-bot.js`
+- Events are registered in `ON_EVENTS` in `src/auto-me-bot.js`
+
+## Documentation
+
+- `docs/index.md` — Overview, `docs/install.md` — Installation, `docs/config.md` — Config reference, `docs/examples.md` — Examples, `docs/handlers/*.md` — Individual handlers
+
+### When to Update
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)